### PR TITLE
2020-07-01 data 

### PR DESCRIPTION
**Pull request by richard-thonig**
data for the second 2020 update.

### CODE REVIEW

File: csp-guru/README.md [hunk 1]
```diff
@@ -14,20 +14,20 @@ The licence of this data allows academic and all non-commercial use if you give
...
 
 ### New Columns
-OpenCSP contains 50 additional columns compared to the last csp.guru version. We added a lot of technical information from the NREL/Solarpaces database. The LCOE and other economics calculations has been made more transparent by adding columns on intermediary results including specific cost and currency deflation.
+OpenCSP version of CSP.guru contains 50 additional columns compared to the csp.guru[2019-01-01] version. We added a lot of technical information from the NREL/Solarpaces database. The LCOE and other economics calculations has been made more transparent by adding columns on intermediary results including specific cost and currency deflation.
```
Comment (timtroendle) [inline csp-guru/README.md:20]: This probably should not be called "OpenCSP version of CSP.guru", should it?

File: csp-guru/README.md [hunk 2]
```diff
@@ -14,20 +14,20 @@ The licence of this data allows academic and all non-commercial use if you give
 For academic and professional use (presentations, journal articles, reports, etc): Please cite the original source, the paper [Lilliestam et al. (2020)](https://doi.org/10.1080/15567249.2020.1773580). Please also link to www.csp.guru if possible.
 
 ## Changes in this version of openCSP
```
[the remark this answers is not in the document]
Comment (timtroendle) [reply csp-guru/README.md:16]: Probably it's a good idea to run a global "search and replace" in the folder to make sure there are traces of openCSP only where appropriate.

File: csp-guru/README.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
-# openCSP
-An open-source database of all concentrating solar power plants of the world for energy modellers and analysts.
+# CSP.guru
+An open-source database of all concentrating solar power plants of the world for energy modelers and analysts. This version of CSP.guru reflects the status of CSP projects as of July, 1st 2020.
```
Comment (timtroendle) [inline csp-guru/README.md:2]: I'd write this as "1 July 2020".

**Review by timtroendle — APPROVED** [review]
Looks good! Just a few typos to be handled.

File: csp-guru/README.md (hunk 3 above)
Comment (richard-thonig) [reply csp-guru/README.md:2]: done.

File: csp-guru/README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (richard-thonig) [reply csp-guru/README.md:16]: yeah, it was more: I halfway renamed it the last time - but apperently there is not openCSP, but only csp.guru appendend by the openCSP project. Whatever. Thx for pointing it out again.

[the remark this answers is not in the document]
Comment (richard-thonig) [reply csp-guru/README.md:16]: did the search. thx.

**Comment by timtroendle** [comment]
Somehow most typos I mentioned in my review aren't fixed. Did you do this intentionally, or did they get lost somehow?

**Comment by timtroendle** [comment]
OK actually I see now that on master things are correct. No idea how you did that, but the main thing is it worked. 🥇 :)

**Pull request merged**